public_html/wp-content/themes/pattern-directory/css/components/_category-menu.scss

```diff
@@ -0,0 +1,89 @@
+$grid-unit: 1.5rem;
+$font-weight: 500;
+$font-size: 14px;
```

Comment:
I'll move the `$grid-unit` since it's very useful, but the other 2 are probably not useful.

Reply:
In `_structure.scss` there's a `$gutter-default` property. I used that instead. It was set to `16px` so I updated it to `1.5rem` which I got from the pattern grid and since it doesn't have many references yet in the theme its a safe-ish change.
 I think we should update all gap/gutter references to use it moving forward.

Reply:
I still don't think we need local variables in these files - you're only using border-radius once, and if we want to keep the assorted font-sizes reusable let's make a set in [`settings/_typography.scss`](https://github.com/WordPress/pattern-directory/blob/trunk/public_html/wp-content/themes/pattern-directory/css/settings/_typography.scss). I'd rather not use any local variables because they aren't actually scoped to the file, if you use `$font-size` in another file, it will work, but you have no good way of knowing which font size it's using.

Reply:
Good call about the $gutter property, we can also make `$gutter-small` etc, as we need.

Reply:
> I still don't think we need local variables in these files - you're only using border-radius once, and if we want to keep the assorted font-sizes reusable let's make a set in [`settings/_typography.scss`](https://github.com/WordPress/pattern-directory/blob/trunk/public_html/wp-content/themes/pattern-directory/css/settings/_typography.scss). I'd rather not use any local variables because they aren't actually scoped to the file, if you use `$font-size` in another file, it will work, but you have no good way of knowing which font size it's using.

Very fair. I'll pull them out.